server/src/main/scala/org/http4s/server/middleware/DefaultHead.scala

```diff
@@ -16,14 +16,10 @@ import cats.implicits._
 object DefaultHead {
   def apply[F[_]: Functor, G[_]](http: Http[F, G])(implicit F: MonoidK[F]): Http[F, G] =
     Kleisli { req =>
-      req.method match {
-        case Method.HEAD => {
-          (http <+> headAsTruncatedGet(http))(req)
-        }
-        case _ => {
-          http(req)
-        }
-      }
+      if (req.method == Method.HEAD)
```

Comment:
I might like the case better here.  I'm unsure whether there's a bytecode difference.

Reply:
Definitely prefer the case statement, but agree with removing the extra bracket.